config/reader.go

```diff
@@ -50,7 +51,9 @@ func loadEnv() (*Product, error) {
 	productConfig := &Product{
 		TFVars:  os.Getenv("ENV_TFVARS"),
 		Product: os.Getenv("ENV_PRODUCT"),
+		Module:  os.Getenv("ENV_MODULE"),
```

Comment:
sorry google kept cutting me out of the presentation yesterday. Am assuming that this will be set from the .env file? 
Can we add some docs/comments around, what values are acceptable for this field? is this the module sub folder names - like airgap/bastion/k3s/rke2 at this time?

Reply:
Yes sure I still have to add the docs, I keep forgetting about it.. This will be set as ENV_MODULE on the .env file and for now it should be airgap, which is the tfmodule name